src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/DefaultSwaggerParameters.java

```diff
@@ -3,13 +3,15 @@
 /** Holds the default values for the parameters to run test-requests in Swagger. */
 public class DefaultSwaggerParameters {
 
-  public static final String BBOX = "8.625,49.3711,8.7334,49.4397";
+  public static final String BBOX = "8.67,49.39,8.71,49.42";
   public static final String HIGHWAY_KEY = "highway";
   public static final String BUILDING_KEY = "building";
-  public static final String TYPE_FILTER = "type:way";
-  public static final String HIGHWAY_FILTER = "highway=residential";
-  public static final String BUILDING_FILTER = "building=*";
-  public static final String HOUSENUMBER_FILTER = "type:node and \"addr:housenumber\"=*";
+  public static final String GENERIC_FILTER = "type:way and natural=*";
+  public static final String HIGHWAY_FILTER = "type:way and highway=residential";
+  public static final String HIGHWAY_FILTER2 = "type:way and highway=*";
+  public static final String BUILDING_FILTER = "geometry:polygon and building=*";
+  public static final String BUILDING_FILTER2 = "geometry:polygon and building=house";
+  public static final String HOUSENUMBER_FILTER = "type:node and \"addr:housenumber=*\"";
```

Comment:
This filter doesn't work:
```bash
% curl -X POST http://localhost:8082/elements/count/ratio -d 'bboxes=8.67,49.39,8.71,49.42' -d 'filter=geometry:polygon and building=*' -d 'filter2=type:node and "addr:housenumber=*"' -d 'format=json' -d 'time=2014-01-01/2017-01-01/P1Y'
{
  "timestamp" : "2021-02-05T21:39:53.503188",
  "status" : 400,
  "message" : "Invalid filter syntax. Please look at the additional info and examples about the filter parameter at https://docs.ohsome.org/ohsome-api. Detailed error message: line 1, column 35: whitespaces, EQUALS (=), NOT_EQUALS (!=) or in expected, EOF encountered.",
  "requestUrl" : "http://localhost:8082/elements/count/ratio"
}
```
This works:
```suggestion
  public static final String HOUSENUMBER_FILTER = "type:node and addr:housenumber=*";
```

Reply:
oops, I overlooked this one while rebasing the branch :see_no_evil: , this bug was actually just recently already fixed in #112 ([comment](https://github.com/GIScience/ohsome-api/pull/112/files#r570749196)). it's fixed now